lib/ethon/easy/options.rb

```diff
@@ -20,6 +20,15 @@ def escape?
         @escape
       end
 
+      def multipart=(b)
+        @multipart = b
+      end
+
+      def multipart?
+        return false if !defined?(@multipart) || @multipart.nil?
```

Comment:
I don't think the `!defined?(@multipart)` check is necessary here.

Reply:
Actually the easiest way to implement this would be:

```ruby
def multipart?
  !!@multipart
end
```

Reply:
Sure, I can also make that change.

Reply:
(I just modeled it after the escape function above it)